src/components/TagBadges/TagsNew.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+
+const TagsNew = () => {
+  const [tags, setTags] = useState<any[]>([])
+  const addTags = (e: any) => {
+    if (e.target.value !== '') {
+      setTags([...tags, e.target.value])
+      e.target.value = ''
+    }
+    console.log(e)
+    console.log(tags)
+  }
+  const removeTags = (indexToRemove: number) => {
+    setTags([...tags.filter((_, index) => index !== indexToRemove)])
+  }
+  return (
+    <>
```

Comment:
We already have a working tags component in my pr dropdown. If you want to implement yours, make the functionality so that it works with the backend: when user starts adding a skill, there should be a suggestion coming from the backend that a user can click, which then adds the tag to the form. After that you can open a separate pr for that. But for now, you should remove this component, because we don't want to take steps backwards in the building process of this app.